media/filters/demuxer_manager.cc

```diff
@@ -26,7 +26,10 @@
 #endif  // BUILDFLAG(ENABLE_HLS_DEMUXER)
 
 #if BUILDFLAG(ENABLE_FFMPEG)
-// ProgressiveDemuxer is enabled when use_starboard_media=true and media_use_ffmpeg=false.
+// The ProgressiveDemuxer is enabled when use_starboard_media=true and
```

Comment:
nit: The ProgressiveDemuxer is enabled when use_starboard_media=true and media_use_ffmpeg=false. `Cobalt enables ProgressiveDemuxer by default.` This can be overridden by the BASE_FEATURE CobaltProgressivePlayback, which will disable the ProgressiveDemuxer when false.